store/subscription.go

```diff
@@ -298,10 +303,9 @@ func (s *subscription) insert(ctx context.Context, events []*model.Transfer) (er
 	}
 
 	// Add diff in total balances
-	for _, sub := range subs {
-		addr := gethCommon.BytesToAddress(sub.Address)
-		for token, addrs := range addrDiff {
-			d, ok := addrs[addr]
+	for token, addrs := range addrDiff {
+		for addr, d := range addrs {
+			sub, ok := allSubs[addr]
 			if !ok {
 				continue
```

Comment:
Print a warming here. It should not be happen.

Reply:
done